debian/control

```diff
@@ -25,7 +25,7 @@ Build-Depends: debhelper (>= 10),
                dpdk-dev (>= 21.11.2),
                libisal-crypto-dev,
                libcunit1-dev,
-               golang-go (>= 1.18),
+               golang-go (>= 1.21),
```

Comment:
This has come up before and got dropped but I think the golang-go version on debian uses an epoch so you need to specify the version as I've indicated here.
```suggestion
               golang-go (>= 2:1.21),
```

Reply:
https://packages.debian.org/bookworm/golang-go for the upstream package, and yes it does have/require an epoch number.